pulseapi/profiles/serializers.py

```diff
@@ -26,11 +26,13 @@ class UserProfileSerializer(serializers.ModelSerializer):
     """
     user_bio = serializers.CharField(
         max_length=140,
-        required=False
+        required=False,
+        allow_blank=True,
     )
     custom_name = serializers.CharField(
         max_length=70,
-        required=False
+        required=False,
+        allow_blank=True,
     )
     is_group = serializers.BooleanField()
```

Comment:
Front end UI doesn't have a field to toggle `is_group` (I think this is an admin only control that's only visible on Django dashboard?). Can serialize do something like "if the value is `null` or the `is_group` key isn't presented, keep the `is_group` value in db unchanged" ?

Reply:
> Front end UI doesn't have a field to toggle is_group (I think this is an admin only control that's only visible on Django dashboard?)

Yes. It's just a moderator/admin feature we'll use via Django, for when staff wants to create a profile for something like Ford Foundation. No real functionality now, but useful down the road.